www/javascript/site-content-slider.js

```diff
@@ -113,6 +117,11 @@ YAHOO.util.Event.onDOMReady(function ()
 
 	// }}}
 
+	SiteContentSlider.sliders = [];
+	SiteContentSlider.getSlider = function(index) {
```

Comment:
This method seems redundant. You can just use array access instead of a method:
```javascript
var my_slider = SiteContentSlider.sliders[0];
// vs
var my_slider = SiteContentSlider.getSlider(0);
```